src/events.ts

```diff
@@ -0,0 +1,194 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as express from 'express';
+import {
+  BackgroundEvent,
+  CloudFunctionsContext,
+  CloudFunctionsResource,
+  CloudEventsContext,
+} from './functions';
+import {
+  isBinaryCloudEvent,
+  isStructuredCloudEvent,
+  getBinaryCloudEventContext,
+} from './cloudevents';
+
+/**
+ * Mapping between background event types and CloudEvent types.
+ */
+const typeBackgroundToCloudEvent: Record<string, string> = {
```

Comment:
Use UPPER_CASE for constants.

`BACKGROUND_TO_CLOUDEVENT_EVENT_TYPE_MAP`

https://google.github.io/styleguide/jsguide.html#naming-constant-names